### PR TITLE
Input type "number" is not styled properly

### LINKED ISSUE #36164: input type "number" is not styled properly
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

If you use input type "text" everything is nice, but if you use input type "number", it's not styled properly. So, we should edit CSS for type "number" as well, it must be same like input type "text"
Check screenshot:
type_number

### Expected behavior

Same styling for input type "text" and type "number".
Module developers can use input type "number", so we should make it nice like "text".

### Steps to reproduce

Use <input type="number"> and compare with <input type="text"> in html in BO for example and you will see results like on my screenshot.

### PrestaShop version(s) where the bug happened

8.1.5

### PHP version(s) where the bug happened

8.1

### If your bug is related to a module, specify its name and its version

_No response_

### Your company or customer's name goes here (if applicable).

openservis.cz

### PR DESCRIPTION
**Pull request by ShaiMagal**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Check issue #36164 
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | 
| UI Tests          | https://github.com/florine2623/testing_pr/actions/runs/9694646890 ✅ 
| Fixed issue or discussion?     | Fixes #36164 
| Related PRs       | 
| Sponsor company   | https://www.openservis.cz/

### CODE REVIEW

File: admin-dev/themes/default/scss/partials/_forms.scss [hunk 1]
```diff
@@ -70,6 +70,7 @@ input[type="color"]:focus,
 
 textarea:hover,
 input[type="text"]:hover,
+input[type="number"]:hover,
```
Comment (Hlavtox) [inline admin-dev/themes/default/scss/partials/_forms.scss:73]: Duplicate hard

File: admin-dev/themes/default/scss/vendor/bootstrap-sass/_forms.scss [hunk 2]
```diff
@@ -98,6 +98,7 @@ output {
 // select
 // textarea
 // input[type="text"]
+// input[type="number"]
```
Comment (Hlavtox) [inline admin-dev/themes/default/scss/vendor/bootstrap-sass/_forms.scss:101]: Duplicate hard

**Review by mflasquin — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Review by kpodemski — CHANGES_REQUESTED** [review]
Could you provide a simple module to allow QA to perform tests?

**Comment by ShaiMagal** [comment]
@kpodemski OK, attaching.
[shaim_number_test.zip](https://github.com/user-attachments/files/15899049/shaim_number_test.zip)
Untitled

**Comment by ShaiMagal** [comment]
@kpodemski Here
[shaim_number_test.zip](https://github.com/user-attachments/files/15899104/shaim_number_test.zip)

**Review by kpodemski — APPROVED** [review]

**Review by Hlavtox — APPROVED** [review]

**Comment by ps-jarvis** [comment]
Hi, thanks for this contribution!

I found some issues with the Pull Request description:

* The `How to test` shouldn&#039;t be empty. ([Read explanation](https://devdocs.prestashop-project.org/9/contribute/contribution-guidelines/pull-requests/#how-to-test))

Would you mind having a look at it? This will help us understand how interesting your contribution is, thank you very much!

(Note: this is an automated message, but answering it will reach a real human)

**Review by florine2623 — COMMENTED** [review]
Hello @ShaiMagal ,

I installed your module in a shop with your PR and in another shop (fresh install); but I see no difference between the 2 inputs.

In both cases, I have :
Screenshot 2024-06-27 at 12 13 46

Am I missing something ?
Waiting for your feedback ^^

**Comment by ShaiMagal** [comment]
@florine2623 
If we only change SCSS files it's without impact, there is necessary to compile SCSS to CSS.
It is in file theme.css, but I don't know how is it realized while installing fresh PS. So compilacted to QA I think:

But maybe I am missing something.

**Comment by florine2623** [comment]
@ShaiMagal ,
I understand your point.
In the PR, the design is not correct. It is not what you put in the issue.

The borders should be rounded and grey, as it is in your issue.

With the module and the PR installed, I have black borders and angled corners.

**What I should have :**
test


**What I have :**
Screenshot 2024-07-02 at 11 14 43

**Review by tleon — APPROVED** [review]

**Comment by kpodemski** [comment]
@florine2623 Dominik forgot to put `$this->bootstrap = true;` to his module, correct module attached below.
[shaim_number_test 2.zip](https://github.com/user-attachments/files/16390000/shaim_number_test.2.zip)

**Review by florine2623 — APPROVED** [review]
Hello @ShaiMagal and @kpodemski !

Thanks for the update !

Indeed, with the latest module it works as expected.

Before :
Screenshot 2024-08-01 at 16 02 23


After :
Screenshot 2024-08-01 at 16 08 38

It is QA ✅ 


Thanks!

**Comment by ShaiMagal** [comment]
@kpodemski uh, sorry, my fault ... missing line. Thank you for correction :)

**Pull request merged**